pallets/pool-system/src/lib.rs

```diff
@@ -40,7 +40,6 @@ use orml_traits::{
 	Change,
 };
 pub use pallet::*;
-use polkadot_parachain::primitives::Id as ParachainId;
```

Comment:
It looks like all usages of `polkadot_parachain` have been removed for the `pool-systems` pallet.  I think we can remove from pallet `Cargo.toml`.

Reply:
Ah yes, I thought about this last night during my sleep xD thanks for the catch!

Reply:
Done in https://github.com/centrifuge/centrifuge-chain/pull/1340/commits/0bf232376e5723ed78f3eed59297f20def740a8f ✅